### ISSUE TITLE
Update Ganga example

### ISSUE THREAD

**Issue by mesmith75** [lhcb/first-analysis-steps#188]
What is the plan for the tutorial with DaVinci and the grid? Are you now recommending lb-run, in which case can we update the Ganga part to use GaudiExec?

**Comment by alexpearce** [comment]
I think these lessons don't use `lb-dev` with Ganga. But now that I think about it, perhaps that doesn't matter.

Is it now recommended that `GaudiExec` should always be used for submitting jobs with Gaudi-based applications, regardless of whether there's a local `lb-dev` directory to be used?

**Comment by mesmith75** [comment]
@alexpearce Yes it is recommended to use GaudiExec. You can use prepareGaudiExec(app,ver) which will make it for you if you haven't already got a local copy.

**Comment by mesmith75** [comment]
Actually, would it be feasible to have a lesson/page for Ganga with more in depth examples? I would be happy to create one...

**Comment by alexpearce** [comment]
Absolutely!

Feel free to break things up across two or three lessons if it starts to get a bit long.

**Comment by henryiii** [comment]
I would love to see a page that explains how to set up Ganga to run locally too,  since the documentation (or a TWiki page) says to run it locally several times for monitoring jobs, but there does not seem to be any current explanation of how to get that set up to talk to the correct user in the grid.  Maybe that would fit better developed it, due to the set of pages, that would not just be for developers.  Maybe that would fit better in DevelopKit, due to the set of pages on set up, but that would not just be for developers.

**Comment by henryiii** [comment]
I would love to see a page that explains how to set up Ganga to run locally too,  since the documentation (or a TWiki page) says to run it locally several times for monitoring jobs, but there does not seem to be any current explanation of how to get that set up to talk to the correct user in the grid.  Maybe that would fit better developed it, due to the set of pages, that would not just be for developers.  Maybe that would fit better in DevelopKit, due to the set of pages on set up, but that would not just be for developers.
